### DISCUSSION TITLE
Is there anyway to change the UID/GID associated with postgres user/group?

### DISCUSSION THREAD

**Discussion by kevdogg** [prodrigestivill/docker-postgres-backup-local#83] [Q&A] [answered]
I was wondering if through a Dockerfile or other means if there was a way to change the UID/GID mappings of the postgres user/group?

**Comment by prodrigestivill** [comment, marked as answer]
Partly related with #29.
But this is a generic Docker+Alpine/Debian question, please refer to those projects if you want to perform this changes.
